src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -222,6 +224,21 @@ public void testIsCommentType() {
                 .isFalse();
     }
 
+    @ParameterizedTest
+    @ValueSource(ints = {
+        TokenTypes.SINGLE_LINE_COMMENT,
+        TokenTypes.BLOCK_COMMENT_BEGIN,
+        TokenTypes.BLOCK_COMMENT_END,
+        TokenTypes.COMMENT_CONTENT,
+    })
+    public void testIsCommentTypeAst(int type) {
+        final DetailAstImpl ast = new DetailAstImpl();
```

Comment:
Can we avoid pure Unit test ? Even it is until, it is better to cover it by full functional test

Reply:
I think it is impossible for such util methods

Reply:
nevermind, I excluded changes in TokenUtil from this PR, it is not needed